src/components/Themes/index.js

```diff
@@ -1,5 +1,6 @@
 export default {
   red: "redTheme",
+  blue: "blueTheme",
   neonCarrot: "neonCarrotTheme",
   greyishBlue: "greyishBlueTheme",
   lightBlue: "lightBlueTheme",
```

Comment:
just my personal preference, and feel free to disregard but can we sort this alphabetically?

Reply:
Yes